src/components/v2/Table/index.tsx

```diff
@@ -1,35 +1,17 @@
 /** @jsxImportSource @emotion/react */
-import React, { useMemo, useState } from 'react';
+import React from 'react';
+import { uid } from 'react-uid';
```

Comment:
I think we should avoid using this library. In this case we are passing cells to it. If the data in the cells change this can cause rendering issues because the key will also change

Reply:
Agreed, removed.